### PR TITLE
move to blink

### PR DESCRIPTION
**Pull request by amaziagur**

### CODE REVIEW

**Review by rockem — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
     <groupId>io.ticktok</groupId>
     <artifactId>ticktok-java-client</artifactId>
-    <version>0.2.0</version>
+    <version>0.4.1</version>
```
Comment (rockem) [inline pom.xml]: 4.1 ?

**Review by rockem — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (amaziagur) [reply pom.xml]: heheh, opps

Comment (amaziagur) [reply pom.xml]: fixed

File: pom.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
     <groupId>io.ticktok</groupId>
     <artifactId>ticktok-java-client</artifactId>
-    <version>0.2.0</version>
+    <version>0.3.0</version>
```
Comment (rockem) [inline pom.xml]: Amazia, this is a patch with no added functionality, it should be 0.2.1

**Review by rockem — CHANGES_REQUESTED** [review]

**Review by rockem — APPROVED** [review]

**Pull request merged**